Anytype/Sources/PresentationLayer/Auth/Auth/AuthView.swift

```diff
@@ -63,7 +65,7 @@ struct AuthView: View {
             .multilineTextAlignment(.center)
             .padding(.horizontal, UIDevice.isPad ? 85 : 38)
             .environment(\.openURL, OpenURLAction { url in
```

Comment:
What is it?

Reply:
You override system openURL?

Reply:
I not understand goal for this hard decision.
You can read url from view model like `.safariSheet(url: $model.safariUrl, preferredColorScheme: .dark)`